<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTableForm</class>
 <widget class="QWidget" name="DataTableForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="dataTbl">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>21</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>columnX</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>columnY</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>251</width>
     <height>43</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="saveBtn">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadBtn">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
